MainClass.cs

```diff
@@ -197,7 +197,8 @@ public static int Main(string[] args)
 
             // All done! Write it out!
 
-            string final_path = Path.Combine(options.OutputDir, String.Format ("{0}-{1}.ckan", mod.identifier, mod.version));
+            var version_filename = mod.version.ToString().Replace(':','-'); 
+            string final_path = Path.Combine(options.OutputDir, $"{mod.identifier}-{version_filename}.ckan");
```

Comment:
My compiler cries at `$"..."`, which I'm assuming is something cool that I would like to use, but can't. :)